jupyterhub/templates/_helpers-names.tpl

```diff
@@ -74,20 +74,23 @@
     {{- include "jupyterhub.fullname.dash" . }}hub
 {{- end }}
 
-{{- /* hub Secret */}}
-{{- define "jupyterhub.hub-secret.fullname" -}}
+{{- /* hub-existing-secret Secret */}}
+{{- define "jupyterhub.hub-existing-secret.fullname" -}}
```

Comment:
Does it make sense to have a helper that just maps exactly onto a single value?

Reply:
I'd like to provide that no matter what for consistency. These helpers are accessible from parent Helm charts. And, with nameOverride and fullnameOverride, they can be important to arrive at the right resource name.

I have a feeling that that always referencing resources using a named template is a helpful practice to stick to.